src/main/scala/io/iohk/ethereum/db/storage/BlockBodiesStorage.scala

```diff
@@ -23,6 +24,12 @@ class BlockBodiesStorage(val dataSource: DataSource) extends KeyValueStorage[Blo
   override def valueDeserializer: (IndexedSeq[Byte]) => BlockBody =
     (encodedBlockBody: IndexedSeq[Byte]) => rlpDecode[BlockBody](encodedBlockBody.toArray)
 
+  override def get(key: BlockBodyHash): Option[BlockBody] = if (key == Config.Blockchain.genesisBlockHeader.hash) {
```

Comment:
storage will be changed when @AlanVerbner PR (https://github.com/input-output-hk/etc-client/pull/51) is merged